.github/workflows/test.yml

```diff
@@ -75,9 +75,11 @@ jobs:
       #              run test suite   
       #----------------------------------------------
       - name: Run tests
+        env:
+          SYNAPSE_ACCESS_TOKEN: ${{ secrets.SYNAPSE_ACCESS_TOKEN }}
```

Comment:
Quick question, this `SYNAPSE_ACCESS_TOKEN` env variable has to do with the Synapse service account?

Reply:
Yup!